### PR TITLE
Debbie's revised pull request about SPARQL endpoints

### PR DESCRIPTION
**Pull request by tombaker**

### CODE REVIEW

File: docs/D2695955.md [hunk 1]
```diff
@@ -225,3 +225,21 @@ Note: Hover over a code to see its URI.  Click on a code to visit its full defin
 
 ## [A:](http://asn.desire2learn.com/resources/S2696024) Creating Linked Data applications
 * [B:](http://asn.desire2learn.com/resources/S2696092) Storing RDF data
+    * [C:] Understands a SPARQL endpoint as an Application Programming Interface (API) for providing responses to SPARQL queries in various formats.
+        * [D:] Uses DBPedia's SNORQL program and interface for retrieving data via SPARQL queries.
+    * [C:] Knows directories, such as Linked Open Vocabularies, for discovering SPARQL endpoints.
+    * [C:] Understands that triplestores usually provide a SPARQL endpoint.
+    * [C:] Understands the difference between a triplestore (which is analogous to a MYSQL database manager) and an endpoint (which may or may not also be a triplestore).
+    * [C:] Is aware of both commercially available and open-source triplestores.
```
Comment (tombaker) [inline docs/D2695955.md]: Perhaps too vague?

Comment (kaiec) [inline docs/D2695955.md]: I am not sure this is needed. Too muc detail.

File: docs/D2695955.md [hunk 2]
```diff
@@ -225,3 +225,21 @@ Note: Hover over a code to see its URI.  Click on a code to visit its full defin
...
+    * [C:] Is aware that a triplestore may or may not support SPARQL or SPARQL-like queries.
+    * [C:] Is aware of how triplestores handle a graph name unique in the context of that dataset or an unnamed graph (e.g., explicitly specifying FROM NAMED in your query).
+    * [C:] Understands how named graphs can designate specific sets of triples in a triplestore.
+    * [C:] Understands the history and development of triple stores and endpoints and how they complimentarily aided in the support and uptake of SPARQL.
+    * [C:] Understands the role of the SPARQL processor working in conjunction with the triplestore or endpoint to process and retrieve data.
+    * [C:] Understands using the SERVICE keyword to query remote RDF data by pointing it to a SPARQL endpoint directly, obviating the need to point to an RDF file somewhere (or at a service delivering the equivalent of an RDF file).
+    * [C:] Understands how using SERVICE differs from not using SERVICE.
```
Comment (kaiec) [inline docs/D2695955.md]: Let's remove this.

File: docs/D2695955.md [hunk 3]
```diff
@@ -225,3 +225,21 @@ Note: Hover over a code to see its URI.  Click on a code to visit its full defin
...
+    * [C:] Understands how SERVICE allows you to embed a query within a query.
```
Comment (tombaker) [inline docs/D2695955.md]: Would delete.

File: docs/D2695955.md [hunk 4]
```diff
@@ -225,3 +225,21 @@ Note: Hover over a code to see its URI.  Click on a code to visit its full defin
...
+    * [C:] Understands how the SERVICE keyword allows you to get at otherwise siloed data.
```
Comment (tombaker) [inline docs/D2695955.md]: Delete...

File: docs/D2695955.md [hunk 5]
```diff
@@ -225,3 +225,20 @@ Note: Hover over a code to see its URI.  Click on a code to visit its full defin
...
+    * [C:] Is aware that a triplestore may or may not support SPARQL or SPARQL-like queries.
+    * [C:] Is aware of how triplestores handle a graph name unique in the context of that dataset or an unnamed graph (e.g., explicitly specifying FROM NAMED in your query).
+    * [C:] Understands how named graphs can designate specific sets of triples in a triplestore.
+    * [C:] Understands the history and development of triple stores and endpoints and how they complimentarily aided in the support and uptake of SPARQL.
+    * [C:] Understands the role of the SPARQL processor working in conjunction with the triplestore or endpoint to process and retrieve data.
+    * [C:] Understands using the SERVICE keyword to query remote RDF data by pointing it to a SPARQL endpoint directly, obviating the need to point to an RDF file somewhere (or at a service delivering the equivalent of an RDF file).
+    * [C:] Understands how using SERVICE differs from not using SERVICE...
+    * [C:] Understands how SERVICE allows you to embed a query within a query.
+    * [C:] Understands how the SERVICE keyword allows you to get at otherwise siloed data.
```
Comment (kaiec) [inline docs/D2695955.md]: Meaningless comment

**Pull request closed without merging**